src/WebJobs.Script.WebHost/Security/KeyManagement/BlobStorageSasSecretsRepository.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    /// <summary>
+    /// An <see cref="BlobStorageSecretsRepository"/> implementation that uses a SAS connection string to connect to Azure blob storage.
+    /// </summary>
+    public sealed class BlobStorageSasSecretsRepository : BlobStorageSecretsRepository
+    {
+        public BlobStorageSasSecretsRepository(string secretSentinelDirectoryPath, string accountConnectionString, string siteSlotName)
```

Comment:
nit: rename from `accountConnectionString` to `containerUri` or something along those lines.